.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+If you are creating an issue to seek help or support with your
+_usage_ of Scrapy, however, please consider posting instead
+to a more appropriate forum: We support Reddit, IRC, and Stack
+Overflow. Check our Community portal for details:
+https://scrapy.org/community/
+
+Otherwise, feel free to delete the above and make your issue!
+If it's a bug, try to make it reproducible and minimal.
+But don't sweat it. Thanks for contributing to Scrapy! :)
```

Comment:
I'm not sure, maybe it makes sense to remove "But don't sweat it.", and write "try to make it reproducible and minimal if possible" or something like this? It's an idiom that might be not 100% clear for everyone, I already knew it and still needed some time to understand what does it mean here :)

Reply:
Agreed.
Many people here does not have English as first or second language (many depend on google translator and similar) so I think it is better to have a clear and direct text, without any "popular expression"[1] like "don't sweat it"[2].

[1] There is a correct name in the grammar for this kind of stuff. But as you can guess, my English is not good...
[2] I have no idea what it means. Going google it now...